operations/iuf/workflows/configuration.md

```diff
@@ -59,6 +60,16 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
    There are comments at the top of the `product_vars.yaml` file that describe the variables and related details. The following are a few examples of `site_vars.yaml` changes:
     - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
     - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+    - Add a 'system-name' entry to the `default` section. The Scalable Boot Projection Service (SBPS) uses this system name as the first part of the domain name. Do not add if not using SBPS.
+      - See the following page in the CSM documentation:  'operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md'
```

Comment:
We should use a link to this section. Something like..

See the `[Section Title](path/to/section)` procedure for more information.

Reply:
I fixed these up.